addons/s3/utils.py

```diff
@@ -18,7 +20,7 @@ def connect_s3(access_key=None, secret_key=None, node_settings=None):
     if node_settings is not None:
         if node_settings.external_account is not None:
             access_key, secret_key = node_settings.external_account.oauth_key, node_settings.external_account.oauth_secret
-    connection = S3Connection(access_key, secret_key, calling_format=OrdinaryCallingFormat())
+    connection = S3Connection(access_key, secret_key)
```

Comment:
this is already in the current feature branch, may cause a merge conflicts (not sure why GitHub is not complaining)

can you rebase the PR on top of the latest feature branch?

see: https://github.com/CenterForOpenScience/osf.io/blob/feature/s3-improvements/addons/s3/utils.py#L21